src/main/java/slimeknights/tconstruct/library/fluid/FluidTankAnimated.java

```diff
@@ -0,0 +1,11 @@
+package slimeknights.tconstruct.library.fluid;
+
+import slimeknights.mantle.tileentity.MantleTileEntity;
+
+public class FluidTankAnimated extends FluidTankBase<MantleTileEntity> {
+  public float renderOffset;
```

Comment:
Make private with getter and setter. lombok can make those for you if you wish. (yes, I know the code in 1.12 did not, we want to make better code going forwards)

Reply:
k.